### ISSUE TITLE
EXIF data are removed

### ISSUE THREAD

**Issue by artonge** [scaleflex/filerobot-image-editor#368]
When editing a picture, its EXIF data are removed.

I think it is expected for those data to be kept in the new version of the picture.

Here is a simple example:

![Screenshot from 2023-08-14 10-38-02](https://github.com/scaleflex/filerobot-image-editor/assets/6653109/95bc21f9-3094-4cfb-826c-11992f2abe93)

**Comment by AhmeeedMostafa** [comment]
@artonge yes it is expected to lose them, as we already create a new image and we don't consider/care for the exif data as already this is a new image created in any case, so if u need to consider it u should do that from ur side by reading the exif data before (in the getting edited image) and apply it after (in the new edited image)

I'm adding this as possible enhancement but not sure if we will consider it in our features plan or not as it is still possible to be applied from ur side.

**Comment by skjnldsv** [comment]
> I'm adding this as possible enhancement but not sure if we will consider it in our features plan or not as it is still possible to be applied from ur side.

If we find a way and open a PR for it, will you consider it?

**Comment by AhmeeedMostafa** [comment]
@skjnldsv yes but depending on its implementation size (if it will take big size or cause any performance issue we won't otherwise it's fine to consider),
and it should be configured whether to enable/disable it through the properties.

**Comment by artonge** [comment]
Maybe a lead if no lib already exists:
https://getaround.tech/exif-data-manipulation-javascript/
